src/azure-cli/azure/cli/command_modules/lab/validators.py

```diff
@@ -530,60 +518,3 @@ def _single(collection):
 
 def _any(collection):
     return len([x for x in collection if x]) > 0
-
-
-def _get_current_user_object_id(graph_client):
-    from msrestazure.azure_exceptions import CloudError
-    try:
-        current_user = graph_client.signed_in_user.get()
-        if current_user and current_user.object_id:  # pylint:disable=no-member
-            return current_user.object_id  # pylint:disable=no-member
-    except CloudError:
-        pass
```

Comment:
I guess this line tries to silence the exception. However, L125 relies on the exception. The code is buggy.